Resources/Changelog/WhiteChangelog.yml

```
@@ -1081,7 +1081,9 @@ Entries:
   time: '2024-12-22T05:31:40.0000000+00:00'
   url: https://github.com/WWhiteDreamProject/wwdpublic/pull/173
 - author: Warete
-  changes: []
+  changes:
+    - type: Add
```

yamllint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)